apps/api/pages/api/bookings/_get.ts

```diff
@@ -162,6 +162,10 @@ function buildWhereClause(
 
 async function handler(req: NextApiRequest) {
   const { userId, isAdmin, prisma } = req;
+
+  const dateFrom = req.query.dateFrom ? new Date(req.query.dateFrom as string) : undefined;
+  const dateTo = req.query.dateTo ? new Date(req.query.dateTo as string) : undefined;
```

Comment:
Apologies for missing this earlier, I think we need to put these in try catch and return a viable error message instead of the generic 
```
{
    "message": "An error occured while querying the database."
}
```
that you'd get right now if you pass invalid dateFrom and/or dateTo. Wrapping them up in a try catch, or better yet, pull out a reusable function that validates while parsing these inputs would be awesome 🙌

Reply:
Once these changes are in, it should be good to go 🙏

Reply:
Hey @DexterStorey did you manage to get a chance to have a look ^?